frontend/src/pages/AccountSend.vue

```diff
@@ -649,32 +1092,43 @@ function useSendForm() {
 
   return {
     acknowledgeSendRisk,
+    addFields,
     advancedAcknowledged,
     advancedMode,
     balances,
+    batchSendHumanToll,
+    batchSendIsSupported,
+    batchSendSupportedChains,
     connectRedirectTo,
     chainId,
     currentChain,
     humanAmount,
     humanAmountBaseInputRef,
     humanToll,
     humanTotalAmount,
+    isDark,
     isSending,
     isValidForm,
     isValidId,
     isValidRecipientId,
     isValidTokenAmount,
     NATIVE_TOKEN,
     onFormSubmit,
+    onBatchSendFormSubmit,
     paymentLinkParams,
     recipientId,
     recipientIdBaseInputRef,
+    removeField,
+    Sends,
     sendAdvancedButton,
     sendFormRef,
+    sendingString,
     sendMax,
     setHumanAmountMax,
     showAdvancedSendWarning,
     showAdvancedWarning,
+    summaryTotalString,
+    tab,
```

Comment:
This file is was already pretty big at ~700 lines, and now it's over 60% bigger at 1167 lines. As a result this is probably going to be hard to maintain. I don't want to block this PR so we can clean it up separately, but we definitely should have split this up into components, such as:
- `AccountSend.vue` is basically the page layout and any shared data that's needed by both child components, managing the tabs, etc.
- `AccountSendSingle.vue` is the "Send" tab component/logic
- `AccountSendBatch.vue` is the "Batch Send" tab component/logic
- `accountSendUtils.ts` can house shared methods/logic